dss/storage/bundles.py

```diff
@@ -119,3 +120,41 @@ def _latest_version_from_object_names(object_names: typing.Iterator[str]) -> str
             version = current_version
 
     return version
+
+
+def list_available_uuids(replica: str = None,
+                         prefix: typing.Optional[str] = None,
+                         per_page: int = PerPageBounds.per_page_max,
+                         search_after: typing.Optional[str] = None,
+                         token: typing.Optional[str] = None):
+    """Return a list of available uuid's in a given prefix, removes tombstones"""
+    kwargs = dict(bucket=Replica[replica].bucket, prefix=prefix, k_page_max=per_page)
+    if search_after:
+        kwargs['start_after_key'] = search_after
+    if token:
+        kwargs['token'] = token
+
+    storage_handler = Config.get_blobstore_handle(Replica[replica])
+    prefix_iterator = storage_handler.list_v2(**kwargs)  # note dont wrap this in enumerate() it looses the token
+    keys = dict()  # type: dict
+    uuid_list = list()
+    total_keys = 0
+
+    for key, meta in prefix_iterator:
+        uuid, version = key.split('.', 1)
+        if not version.endswith(TOMBSTONE_SUFFIX):
+            search_after = key
+            keys.setdefault(uuid.strip(f'{BUNDLE_PREFIX}/'), []).append(version)
```

Comment:
`uuid.strip(..)` should be replaced with a splice if the bundle prefix is always expected to be present, since `str.split` doesn't work the way one might expect. The [pydoc](https://docs.python.org/3/library/stdtypes.html#str.strip) says it better than I can.

For example:

```
>>> from dss.storage.identifiers import BUNDLE_PREFIX
>>> test_uuid = "bundles/bbbebbbb-2101-4e52-94d9-29eebebebebe"
>>> test_uuid.strip(f'{BUNDLE_PREFIX}/')
'-2101-4e52-94d9-29'
```

Were `uuid` to contain a character in `BUNDLE_UUID` (i.e. `b` or `e`) in the wrong place, unexpected behavior would occur.

Reply:
nice catch! addressed.

Reply:
Has this been addressed? `split` should be used instead of `strip`

Reply:
yeah, the commit history was not up-to-date when your review came in, but its been changed.